metaflow/cli.py

```diff
@@ -818,7 +821,7 @@ def before_run(obj, tags, decospecs):
         decorators._attach_decorators(obj.flow, all_decospecs)
         obj.graph = FlowGraph(obj.flow.__class__)
 
-    obj.check(obj.graph, obj.flow, obj.environment, pylint=obj.pylint)
+    obj.check(obj.graph, obj.flow, obj.environment, linter=obj.linter)
```

Comment:
@savingoyal removing the hardfixing of vars away from any specific linter to a generalized naming.